src/main/java/App.java

```diff
@@ -0,0 +1,2 @@
+package PACKAGE_NAME;public class App {
```

Comment:
Was is das? :D

Reply:
1. Formatter is seems to be off 
2. I guess currently your application is not starting up right?